scripts/cli/tests/0_create_identities.js

```diff
@@ -52,17 +52,15 @@ async function createIdentities(api, accounts, alice) {
 
       // Add CDD Claim with CDD_ID
       for (let i = 0; i < dids.length; i++) {
-        const nonceObj = {nonce: reqImports.nonces.get(alice.address)};
+       
         const cdd_id_byte = (i+1).toString(16).padStart(2,'0');
         const claim = { CustomerDueDiligence: `0x00000000000000000000000000000000000000000000000000000000000000${cdd_id_byte}`};
 
         console.log( `>>>> [add CDD Claim] did: ${dids[i]}, claim: ${JSON.stringify( claim)}`);
-        const transaction = api.tx.identity.addClaim(dids[i], claim, null);
-        const result = await reqImports.sendTransaction(transaction, alice, nonceObj);
-        const passed = result.findRecord('system', 'ExtrinsicSuccess');
-        if (passed) reqImports.fail_count--;
+        const transaction = api.tx.identity.addClaim(dids[i], claim, null); 
+        let tx = await reqImports.sendTx(alice, transaction);
+        if(tx !== -1) reqImports.fail_count--;
```

Comment:
Is there a particular reason we don't bake this line into `sendTx`?

I feel like this is the common case in almost all cases that we want this decrement right after, so it would help readability and correctness to include it in `sendTx`. For the cases where the decrement is unwanted, I think we can use another base function `sendTxMaybeFailing` (or some other naming 🤷) which `sendTx` will then call.

Reply:
I plan to use chai or another testing library in JS to move away from the current method of testing for failures this was only temporary. Also plan to move the scripts over to Typescript in the long run as well.